testing/resources/test-env.tmpl.sh

```diff
@@ -1,6 +1,8 @@
 # Environment variables for system tests.
```

Comment:
You'll need to update the encrypted version of this in secrets.tar.gz, the password is in valentine.